owncloudApp/src/main/java/com/owncloud/android/shares/ui/fragment/PublicShareDialogFragment.kt

```diff
@@ -146,33 +155,43 @@ class PublicShareDialogFragment : DialogFragment() {
      * Get expiration date imposed by the server, if any
      */
     private val imposedExpirationDate: Long
-        get() = if (mCapabilities != null && mCapabilities!!.filesSharingPublicExpireDateEnforced.isTrue) {
+        get() = if (capabilities != null && capabilities!!.filesSharingPublicExpireDateEnforced.isTrue) {
```

Comment:
It'd be better without that != null..

`capabilities?.filesSharingPublicExpireDateEnforced?.isTrue == true`

Reply:
Done